client/my-sites/checkout/upsell-nudge/professional-email-upsell/index.tsx

```diff
@@ -155,7 +155,7 @@ const ProfessionalEmailUpsell = ( {
 		<div>
 			<header className="professional-email-upsell__header">
 				<h3 className="professional-email-upsell__small-title">
-					{ translate( "Hold tight, we're getting your site ready." ) }
+					{ translate( "Hold tight, we're getting things ready." ) }
```

Comment:
Because if the user has bought a domain, `we're getting your site ready ` sounded a bit surprising, IMO.

Reply:
I eventually adjusted this to say `Hold tight, we're getting your domain ready.` when it's a domain only site.